### PR TITLE
1734-lobidOrganisationsLabels

### PR DESCRIPTION
**Pull request by TobiasNx**
Use the proper labels from lobid-organisations.

### CODE REVIEW

File: src/main/resources/alma/fix/item.fix [hunk 1]
```diff
@@ -122,8 +126,11 @@ do list(path:"POR  ", "var": "$i")
...
       prepend("hasItem[].$last.heldBy.id", "http://lobid.org/organisations/")
       append("hasItem[].$last.heldBy.id","#!")
+      copy_field("hasItem[].$last.heldBy.id", "hasItem[].$last.heldBy.label")
       # item id is constructed "http://lobid.org/items/[almaMmsId of the record]:[isil of the Owner]:[almaMmsId of the holding]#!"
       paste("hasItem[].$last.id", "~http://lobid.org/items/","almaMmsId", "~:", "hasItem[].$last.heldBy.isil","~:", "$i.a","~#!", join_char: "")
     end
   end
 end
+
+lookup("hasItem[].*.heldBy.label", "lobidOrgLabels")
```
Comment (blackwinter) [inline src/main/resources/alma/fix/item.fix:136]: Would it make (more) sense to skip unknown labels? (`delete: "true"`)

**Review by blackwinter — APPROVED** [review]
Awesome, thank you! :tada:

**Review by acka47 — APPROVED** [review]
+1

File: src/test/resources/alma-fix/990015048200206447.json [hunk 2]
```diff
@@ -61,16 +61,16 @@
...
     "id" : "http://lobid.org/organisations/ZDB-41-UTBL#!",
-    "type" : [ "Collection" ],
-    "label" : "utb-studi-e-book / Psychologie"
+    "label" : "utb-studi-e-book / Psychologie",
+    "type" : [ "Collection" ]
   }, {
     "id" : "http://lobid.org/organisations/ZDB-41-UTBL11#!",
-    "type" : [ "Collection" ],
-    "label" : "lobid Organisation"
+    "label" : "http://lobid.org/organisations/ZDB-41-UTBL11#!",
```
Comment (TobiasNx) [inline src/test/resources/alma-fix/990015048200206447.json:72]: @blackwinter  see here

**Comment by dr0i** [comment]
Will be deployed next Monday.

File: src/main/resources/alma/index-config.json [hunk 3]
```diff
@@ -99,8 +99,12 @@
             "heldBy" : {
                "properties" : {
                   "label" : {
-                     "index": "false",
-                     "type" : "text"
+                     "type" : "text",
+                     "fields": {
+                        "raw": {
```
Comment (blackwinter) [inline src/main/resources/alma/index-config.json:104]: This is the mapping for doc type `item`, but we need it for `resource`.

Comment (dr0i) [reply src/main/resources/alma/index-config.json:104]: Thx! I will remove `item` as we don't need it and it just make trouble.

**Pull request merged**